aerogear/unifiedpush-operator
# AEROGEAR-9778 - Add resource limits to UPS CRs

JIRA: https://issues.jboss.org/browse/AEROGEAR-9778

I basically added parameters in the Push Server CR to replace the hardcoded values here: https://github.com/aerogear/unifiedpush-operator/pull/62

Verification:
1. Try `oc apply -f push_v1alpha1_unifiedpushserver_cr.yaml` and you should see the resource requests/limits on OpenShift web console just as described on https://issues.jboss.org/browse/AEROGEAR-9765
2. Try `oc apply -f push_v1alpha1_unifiedpushserver_cr_with_resource_limits.yaml` and you should see different values
3. Change the values in the UI and see them reconciled
4. Leave some values empty in the CR and some filled. The defaults should be used for the empty ones, but CR should not be touched.
